features/waarden.feature

```diff
@@ -0,0 +1,194 @@
+      },
+      {
+        "vastgesteldeWaarde": 160000,
+        "waardepeildatum": "2018-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 155000,
+        "waardepeildatum": "2017-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 149000,
+        "waardepeildatum": "2016-01-01"
+      }
+    ]
+    """
+
+  Scenario: Belanghebbende krijgt verlaging van de WOZ-waarde na bezwaar
+    Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde op 01-01-2017
+    En dat bezwaar is afgehandeld met de status "bezwaar afgehandeld, vastgestelde waarde veranderd"
+    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde op 01-01-2018
+    En dat bezwaar heeft de status "bezwaar ingediend"
+    En WOZ-object met objectnummer "082600014669" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2019 | 195.000 euro | R. Geels       | 01-01-2020   | 13-01-2020        |
+      | 01-01-2017 | 152.000 euro | R. Geels       | 01-01-2018   | 03-02-2018        |
+      | 01-01-2018 | 176.000 euro | R. Geels       | 01-01-2019   | 14-01-2019        |
+      | 01-01-2017 | 171.000 euro | R. Geels       | 01-01-2018   | 15-01-2018        |
+      | 01-01-2016 | 164.000 euro | R. Geels       | 01-01-2017   | 16-01-2017        |
+    Als het WOZ object wordt opgevraagd met /wozobjecten/082600014669
+    Dan bevat het antwoord:
+    """
+    "waarden": [
+      {
+        "vastgesteldeWaarde": 195000,
+        "waardepeildatum": "2019-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 176000,
+        "indicatieLopendGeding": true,
+        "waardepeildatum": "2018-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 152000,
+        "waardepeildatum": "2017-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 164000,
+        "waardepeildatum": "2016-01-01"
+      }
+    ]
+    """
+
+  Scenario: WOZ-object heeft nieuwe eigenaar en vorige eigenaar heeft verlaging van de WOZ-waarde na beroep
+    Gegeven WOZ-object met objectnummer "050300024029" heeft met ingang van 01-05-2019 een nieuwe eigenaar R. Krol
+    En de vorige eigenaar W. Jansen heeft beroep ingediend over de WOZ-waarde op 01-01-2018
+    En dat beroep is afgehandeld met de status "uitspraak beroep, vastgestelde waarde veranderd"
+    En de vorige eigenaar W. Jansen heeft hoger beroep aangetekend over de WOZ-waarde op 01-01-2017
+    En dat hoger beroep heeft de status "hoger beroep aangetekend"
+    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde op 01-01-2019
+    En dat bezwaar heeft de status "bezwaar ingediend"
+    En WOZ-object met objectnummer "050300024029" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2018 | 190.000 euro | R. Krol        | 01-01-2019   | 27-08-2020        |
```

Comment:
@fsamwel @rijk weten jullie zeker dat het niet om de waarde peildatum 1-1-2018 op de laatste beschikking van de huidig belanghebbende eigenaar gaat? Jij geeft hier aan dat het gaat om de waarde peildatum 1-1-2018 op de laatste beschikking van de vorige eigenaar staat. Kan goed zijn dat ik het gisteren toch verkeerd heb geïnterpreteerd of verkeerd heb onthouden...

Reply:
fouten in scenario zijn gecorrigeerd, wellicht is het dan wel duidelijk. we kijken per waardepeildatum naar de meest recente datum ingang, en daarbinnen naar de meest recente beschikking